packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -434,11 +434,6 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
       return false;
     }
 
-    if (resultCode != Activity.RESULT_OK) {
```

Comment:
onActivityResult() should look at the resultcode: https://developer.android.com/training/basics/intents/result.html#ReceiveResult.

If we are removing this check, we should handle GoogleSignInResult being null because documentation suggests that it could be.